snippets/readme.md

```diff
@@ -0,0 +1,9 @@
+# Snippets
+
+This folder contains the following files:
+
+- snippets.json
+- feature_snippets_react.json
+- feature_snippets_sdk.json
```

Comment:
the unity ones as well?

Reply:
No, it will only generate these 3 files ( same as what we had in docs repo )